src/transformers/cache_utils.py

```diff
@@ -320,3 +322,77 @@ def reorder_cache(self, beam_idx: torch.LongTensor):
             self.key_cache[layer_idx] = self.key_cache[layer_idx].index_select(0, beam_idx.to(device))
             device = self.value_cache[layer_idx].device
             self.value_cache[layer_idx] = self.value_cache[layer_idx].index_select(0, beam_idx.to(device))
+
+class StaticCache(Cache):
+
+    def __init__(self, config: PretrainedConfig, max_batch_size, device = "cuda:2") -> None:
```

Comment:
```suggestion
    def __init__(self, config: PretrainedConfig, max_batch_size, device = "cpu") -> None:
```
maybe better to init on CPU?

Reply:
That's for debugging for now!